src/map/intif.c

```diff
@@ -543,7 +543,7 @@ int intif_create_party(struct party_member *member,char *name,int item,int item2
 		return 0;
 	nullpo_ret(member);
 
-	WFIFOHEAD(inter_fd,64);
+	WFIFOHEAD(inter_fd, 30+sizeof(struct party_member));
```

Comment:
If we already touch it this should be 6 + NAME_LENGTH + sizeof(struct party_member)
Dont you think so?
Same should be applied to the actual size of the packet at offset 2.